avocado/core/jobdata_compat_36_to_52.py

```diff
@@ -30,7 +30,6 @@ class MuxTree(mux.MuxPlugin):
     Excerpt of MuxTree object in order to make it compatible with 52
     """
     pools = []
-    filters = [None, None]
```

Comment:
The `filters` were used by some version between 36 to 52. Anyway probably that version is not compatible as our selftests don't fail and I tried some non-variant files as well. Let's hope it's safe...